src/booleans.js

```diff
@@ -1,61 +1,133 @@
 function negate(a) {
-  // your code here
+  if (a) {return false}
+  else {return true};
 };
 
 function both(a, b) {
-  // your code here
-};
+  if (a === true && b === true) {
+    return true;}
+    else {
+      return false;}
+  
+  };
 
 function either(a, b) {
-  // your code here
-};
+  if (a === false && b === false) {
+    return false}
+    else {
+      return true;}
+  }
+
 
 function none(a, b) {
-  // your code here
-};
+  if (a === true || b === true) {
+    return false;
+  }
+  else{
+    return true;
+  }
+}
 
 function one(a, b) {
-  // your code here
+  if (a === b) {
+    return false;
+  }
+  else {
+    return true;
+  }
 };
 
 function truthiness(a) {
-  // your code here
+  if (Boolean(a) === true){
+    return true;
+  }
+  else {
+    return false;
+  }
 };
 
 function isEqual(a, b) {
-  // your code here
+  if (a === b) {
+    return true;
+  }
+  else {
+    return false;
+  
+  }
+  
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  if(a > b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  if (a <= b) {
+    return true;
+  }
+  else{
+    return false;
+  }
 };
 
 function isOdd(a) {
-  // your code here
+  if (a % 2 ===1) {
+    return true;
+  } else{
+    return false;
+  }
 };
 
 function isEven(a) {
-  // your code here
+  if (a % 2===0) {
+    return true;
+  } 
+  else {
+    return false;
+  }
 };
 
 function isSquare(a) {
-  // your code here
+  if (Math.sqrt(a) % 1 ===0) {
+    return true;
+  }
+  else {
+      return false;
+    }
 };
 
 function startsWith(char, string) {
-  // your code here
+  if (string.charAt(0) === char)
+  {
+    return true;
+  }
+  else {
+    return false;
+  }
 };
 
 function containsVowels(string) {
-  // your code here
+  if (string.match(/[aeiou]/gi))
```

Comment:
Very well done for using regex :)

This could be similar to the previous ones.
You have the statement: `string.match(/[aeiou]/gi)`
If you straightaway return this statement, it will return an array with the vowel if anything matches. 
If there is a value that means it is thruthy, so somehow you would have to turn this ['a'] (which is what the statement returns) into true.
Here you could take advantage of the double NOT operator.
If you add one !, which is the not operator you would do: `!string.match(/[aeiou]/gi)` this would return the opposite of the value, which would be false if the value is thruthy and true if the value is falsy. 
This is why we need to use the double NOT which would be: `!!string.match(/[aeiou]/gi)` in this case if the value is thruthy, it would return NOTNOTtrue which is true.

`````
return !!string.match(/[aeiou]/gi)
``````